assembly/src/lib.rs

```diff
@@ -4,15 +4,17 @@
 #[macro_use]
 extern crate alloc;
 
+use core::ops;
+
```

Comment:
minor nit: I'd get rid of the blank line after this import.

Reply:
Sounds good! We might want to build a rustfmt.toml with the format settings so these are normalized also in the CI level.

I'll check that when I have some free cycles. Creating an issue to track it